data/lua/core/filesystem.lua

```diff
@@ -0,0 +1,16 @@
+
+--[========[File Handling]========]
+print("Loading filesystem module...")
+
+filesystem.asset_type = {
+    IMAGE = 'images',
+    SOUND = 'sounds',
+    MUSIC = 'music',
+    MAP = 'maps',
+    TRANSLATION = '',
+}
```

Comment:
Are these really the only asset types?  Also, where is this used?  I just did a page search here and it doesn't show `asset_type` as being used by anything.  Also also, is there a reason for TRANSLATION to be an empty string?

Reply:
The asset type can basically be any string, these are just the standard ones. I'm unsure about `TRANSLATION`; I _thought_ I recalled making the po loader use the binary path with an empty string, but I couldn't find where that was in the code. Even if I'm right, perhaps it doesn't make sense for it to be here, since 1) it doesn't really make sense for someone to want to load a translation file and 2) if the add-on happened to use mo files, it wouldn't even work anyway.

So I'll probably delete the `TRANSLATION` one.